src/proc.py

```diff
@@ -56,7 +56,7 @@ def MungeExe(cmd, cwd):
   if exe.endswith('.py'):
     script = Which(exe, cwd, is_executable=False)
     return [sys.executable, script] + cmd[1:]
-  if exe == 'git' or exe == 'gclient':
+  if exe in ('git', 'npm', 'gclient'):
```

Comment:
Instead of special-casing 'npm' here, we could just do `Executable('npm', '.cmd')` above in the check_call invocation. I guess this is OK since npm is used twice (although not on Windows, but still)